src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -0,0 +1,599 @@
+---
+layout: post
+title: Fast playback with video preload
+authors:
+  - beaufortfrancois
+description: |
+  Faster playback start means more people watching your video. That's a known fact. In this article I'll explore techniques you can use to accelerate your media playback by actively preloading resources depending on your use case.
+date: 2017-08-17
+updated: 2020-10-21
+tags:
+  - media
+  - video
+---
+
+Faster playback start means more people watching your video. That's a known
+fact. In this article I'll explore techniques you can use to accelerate your
```

Comment:
Linking to evidence would be more compelling than saying `that's a known fact`

Reply:
@beaufortfrancois

Reply:
Ping!

Reply:
Maybe https://www.digitaltrends.com/web/buffer-rage/